build/release/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.13.1 as builder
+
+MAINTAINER Olaoluwa Osuntokun <laolu@lightning.engineering>
+
+ENV GO111MODULE=auto
+
+RUN apt-get install git
```

Comment:
`git` and `tar` are already installed in the base image. But `zip` is missing, we use that too.
We need to change these two lines into:
`RUN apt-get update && apt-get install -y zip`